docs/index.bs

```diff
@@ -1028,10 +1028,11 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       The <dfn method for="Client"><code>postMessage(|message|, |transfer|)</code></dfn> method *must* run these steps:
 
         1. Let |sourceSettings| be the <a>context object</a>'s <a>relevant settings object</a>.
-        1. Let |destination| be the {{ServiceWorkerContainer}} object whose [=ServiceWorkerContainer/service worker client=] is the <a>context object</a>'s [=Client/service worker client=], or null if no match is found.
-        1. If |destination| is null, <a>throw</a> an "{{InvalidStateError}}" {{DOMException}}.
         1. Let |serializeWithTransferResult| be <a abstract-op>StructuredSerializeWithTransfer</a>(|message|, |transfer|). Rethrow any exceptions.
-        1. Add a <a>task</a> that runs the following steps to |destination|'s [=ServiceWorkerContainer/client message queue=]:
+        1. Let |targetClient| be the [=context object=]'s [=Client/service worker client=], or null if it does not exist (i.e. if the [=/service worker client=] has been [=Handle Service Worker Client Unload|unloaded=]).
```

Comment:
I'm not sure what "or null if it does not exist" means. Have we defined how to check for existence? If so we should link to that. Also, is it ok to perform this existence check on the main thread?

Reply:
@jakearchibald, I pushed a commit to address these points. PTAL.